README.md

```diff
@@ -252,6 +252,13 @@ npm run dev
                 <sub><b>Gonçalo Morais</b></sub>
             </a>
         </td>
+        <td align="center">
```

Comment:
FYI — this readme section was broken for a while and when we're at it, i'll probably remove it because it doesn't show all contributoes. 

Your avatar will still be shown on tiny-helpers.dev though. 

<img width="274" alt="image" src="https://github.com/stefanjudis/tiny-helpers/assets/962099/7aec12f8-eb50-43a3-ac8b-8fc19d8632fd">